config/application.rb

```diff
@@ -14,7 +14,6 @@
 # require "action_cable/engine"
 require "sprockets/railtie"
 require "rails/test_unit/railtie"
-require "elasticsearch/rails/instrumentation"
```

Comment:
Basic instrumentation is already built in Searchkick, no need for explicit require.